### ISSUE TITLE
Ctrl+WinKey pastes text

### ISSUE THREAD

**Issue by nesper8** [Swirt/typertools#14]
I'm someone who uses virtual desktops a lot on Windows, like a lot.

The default key binds for changing virtual desktops are Ctrl+WinKey+Left/Right Arrow

But for some reason, typertools have set Ctrl+WinKey to paste text which is kinda a bit PITA especially when i have the extension is open. Is it possible to disable this?

**Comment by Swirt** [comment]
No, unfortunately I didn't provide for that option. I can only advise you to close the extension panel when you don't use it. 
I'll do something about it in future versions.

**Comment by nesper8** [comment]
Mind I keep this open in case you would take a look into this in the future?

**Comment by mohamedosama12345** [comment]
> No, unfortunately, I didn't provide for that option. I can only advise you to close the extension panel when you don't use it. I'll do something about it in future versions.

are there any plans or ETAs for a near future version? 🌹

**Comment by nesper8** [comment]
Hi any ETA on a fix for this issue?
